### PR TITLE
Update css quick guide approach 

### PR DESCRIPTION
**Pull request by david-ziffity**
## Purpose of this pull request
This pull request (PR) adds a new section to the css quick guide approach about extending module styles.

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.html

whatsnew
Added a new section about [extending module styles](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by david-ziffity** [comment]
@jeff-matthews  Yes it is different. The preceding sections are explaining about overriding or extending the Magneto's default ui component's style. The new one is explaining about overriding a module's styles, Since the module.less file is specific to modules and the file structure is different. But the overriding and extending concepts are same.

**Comment by jeff-matthews** [comment]
Thanks for clarifying.

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 1]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
 
 1. Add your styles for the button component. If the file is left blank, then no styles are applied for the component.
 
+
+### Extend Module's styles {#module_extend}
+
+To extend a Module's styles in your theme:
+
+1. In your theme directory, create Module_Name/web/css/source sub-directory.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
1. In your theme directory, create a `Module_Name/web/css/source` directory.
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 2]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+1. Create a _extend.less file here. The path to it looks like following:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
1. Create an `_extend.less` file in the theme directory. For example:
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 3]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+   ```tree
+   <theme_dir>/
+   ├── <module_dir>/
+   │   ├── web/
+   │   │   ├── css/
+   │   │   │   ├── source/
+               ├──_extend.less
+   ...
+   ```
+1. Add your additional styles in the _extend.less.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
1. Add additional styles in the `_extend.less` file.
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 4]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+
+For example:
+
+To extend Magento_Review module's style, the directory path should be <your_theme_dir>/Magento_Review/web/css/source/_extend.less
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
For example, to extend the Magento_Review module's style, the directory path should be `<your_theme_dir>/Magento_Review/web/css/source/_extend.less`.
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 5]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+   ├── <module_dir>/
+   │   ├── web/
+   │   │   ├── css/
+   │   │   │   ├── source/
+               ├──_extend.less
+   ...
+   ```
+1. Add your additional styles in the _extend.less.
+
+For example:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion

```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 6]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+
+To extend Magento_Review module's style, the directory path should be <your_theme_dir>/Magento_Review/web/css/source/_extend.less
+
+### Override Module's styles {#module_override}
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
### Override module styles
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 7]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
 
 1. Add your styles for the button component. If the file is left blank, then no styles are applied for the component.
 
+
+### Extend Module's styles {#module_extend}
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
### Extend module styles
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 8]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+   ```
+1. Add your additional styles in the _extend.less.
+
+For example:
+
+To extend Magento_Review module's style, the directory path should be <your_theme_dir>/Magento_Review/web/css/source/_extend.less
+
+### Override Module's styles {#module_override}
+
+To override a Module's styles in your theme:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
To override module styles in your theme:
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 9]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+
+1. In your theme directory, create Module_Name/web/css/source sub-directory.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
1. In your theme directory, create a `Module_Name/web/css/source` directory.
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 10]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+1. Create a _module.less file here. The path to it looks like following:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
1. Create a `_module.less` file in the theme directory. For example:
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 11]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+   ```tree
+   <theme_dir>/
+   ├── <module_dir>/
+   │   ├── web/
+   │   │   ├── css/
+   │   │   │   ├── source/
+               ├──_module.less
+   ...
+   ```
+   This file overrides the `_module.less` of the specific module.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
   This file overrides the `_module.less` file of the specific module.
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 12]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+
+1. Add your styles in the _module.less.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
1. Add your styles in the `_module.less` file.
```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 13]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+
+For example:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion

```

File: src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 14]
```diff
@@ -96,6 +96,53 @@ To override the parent theme's styles for buttons in your theme:
...
+
+To override Magento_Review module's style, the directory path should be <your_theme_dir>/Magento_Review/web/css/source/_module.less
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md]:
```suggestion
For example, to override the Magento_Review module's style, the directory path should be `<your_theme_dir>/Magento_Review/web/css/source/_module.less`.
```

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
Tests failed due to markdown linting errors:

```
+ rake test:md
[35mTesting Markdown style with mdl ...[0m
[31mThe Markdown linter detected 3 issue(s)[0m
[33msrc/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md:99: MD012 Multiple consecutive blank lines
src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md:118: MD012 Multiple consecutive blank lines
src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md:141: MD012 Multiple consecutive blank lines

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

**Comment by david-ziffity** [comment]
@jeff-matthews Fixed the errors. Please review it. Thanks for your support.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**